stacks/cloud/README.md

```diff
@@ -8,6 +8,7 @@ If you don't have a domain or can't configure the main DNS server, you can confi
 
 ```
 127.0.0.1   api.fog
+127.0.0.1   api.knot.cloud
```

Comment:
I think `api.knot.cloud` in the instructions below stands for the testing instance deployed on AWS instead of a locally deployed cloud. As this is a quick start guide, we don't want to require the user to bring both stacks up. 

I think you can add a note to explain this right before the step-by-step section. Also, we can rename from `*.fog` to `*.cloud` and explain the stack can be used as a fog instance when deployed with a `connector` addon.

Reply:
Done :smile: